tests/src/Kernel/fixtures/markup_rendering_patterns/content_banner.yml

```diff
@@ -87,7 +87,7 @@ content_banner_gray_background_with_image_without_badges_without_meta_without_li
       'div.bcl-content-banner': 1
       'div.bcl-content-banner.bg-lighter': 1
       'img': 1
-      'article.card > div.row > div.col-md-4.col-lg-3 > img[src="https://placeimg.com/1200/600/tech"]': 1
+      'article.card > div.row > div.bcl-card-start-col > img[src="https://placeimg.com/1200/600/tech"]': 1
```

Comment:
Is this change in size expected?
Before
![image](https://user-images.githubusercontent.com/8488617/185697973-14117a7a-d278-46d2-af0c-2f7472a82093.png)

and now
![image](https://user-images.githubusercontent.com/8488617/185698002-e859b353-b8d1-4c80-9525-8f3c70c33f34.png)

It doesn't match the UX files I found.

Reply:
I had a look in figma, and indeed on a content page the image is supposed to be large.
I don't know on which content type/page this is, but that's probably a mistake
![image](https://user-images.githubusercontent.com/25579221/186116418-0def4927-9dfc-4cc4-9b1e-0225eb45de33.png)

Reply:
This is the consequence of the card refactoring, a parameter is now in place to determine the image size, you can see here  in the content banner demo how you can set the image size:
https://oelibrary.netlify.app/default/?path=/story/compositions-content-banner--default&args=image_size:xl

Reply:
In the second image from francesco is the default size (160x160) which is displayed on the listings pages - https://oelibrary.netlify.app/default/?path=/story/features-person--listing
and the large size (255x255) is displayed on the details pages (content pages) 
https://oelibrary.netlify.app/default/?path=/story/features-person--details

As davide mentioned, we introduced 4 sizes:
- 120*120 - sm
- 160*160 - md (default)
- 255*255 - lg
- 350*350 - xl - no use cases for it now so it does not have a demo